app/views/katello/api/v2/subscription_facet/subscription_status.json.rabl

```diff
@@ -0,0 +1,3 @@
+
+attributes :to_global => :color_code
```

Comment:
the existing method names (in foreman) leave a little to be desired but i'll see what i can do